velox/exec/tests/PlanNodeSerdeTest.cpp

```diff
@@ -478,95 +478,31 @@ TEST_F(PlanNodeSerdeTest, write) {
-      planBuilder.planNode());
-  auto plan = planBuilder
-                  .tableWrite(
-                      tableColumnNames,
-                      aggregationNode,
-                      insertHandle,
-                      false,
-                      connector::CommitStrategy::kTaskCommit)
-                  .planNode();
+  auto plan = planBuilder.tableWrite("targetDirectory").planNode();
   testSerde(plan);
 }
 
 TEST_F(PlanNodeSerdeTest, tableWriteMerge) {
   auto rowTypePtr = ROW({"c0", "c1", "c2"}, {BIGINT(), BOOLEAN(), VARBINARY()});
   auto planBuilder =
       PlanBuilder(pool_.get()).tableScan(rowTypePtr, {"c1 = true"}, "c0 < 100");
-  planBuilder.planNode();
-  auto tableColumnNames = std::vector<std::string>{"c0", "c1", "c2"};
-  auto tableColumnTypes =
-      std::vector<TypePtr>{BIGINT(), BOOLEAN(), VARBINARY()};
-  auto locationHandle = exec::test::HiveConnectorTestBase::makeLocationHandle(
-      "targetDirectory",
-      std::optional("writeDirectory"),
-      connector::hive::LocationHandle::TableType::kNew);
-  auto hiveInsertTableHandle =
-      exec::test::HiveConnectorTestBase::makeHiveInsertTableHandle(
-          tableColumnNames, tableColumnTypes, {"c2"}, locationHandle);
-  auto insertHandle =
-      std::make_shared<core::InsertTableHandle>("id", hiveInsertTableHandle);
-  core::TypedExprPtr inputField =
-      std::make_shared<const core::FieldAccessTypedExpr>(BIGINT(), "c0");
-  auto callExpr = std::make_shared<const core::CallTypedExpr>(
-      BIGINT(),
-      std::vector<core::TypedExprPtr>{inputField},
-      "presto.default.min");
-  std::vector<std::string> aggregateNames = {"min"};
-  std::vector<core::AggregationNode::Aggregate> aggregates = {
-      core::AggregationNode::Aggregate{callExpr, nullptr, {}, {}}};
-  auto aggregationNode = std::make_shared<core::AggregationNode>(
-      core::PlanNodeId(),
-      core::AggregationNode::Step::kPartial,
-      std::vector<core::FieldAccessTypedExprPtr>{},
-      std::vector<core::FieldAccessTypedExprPtr>{},
-      aggregateNames,
-      aggregates,
-      false, // ignoreNullKeys
-      planBuilder.planNode());
-  auto plan = planBuilder
-                  .tableWrite(
-                      tableColumnNames,
-                      aggregationNode,
-                      insertHandle,
-                      false,
-                      connector::CommitStrategy::kTaskCommit)
+  auto plan = planBuilder.tableWrite("targetDirectory")
                   .localPartition(std::vector<std::string>{})
                   .tableWriteMerge()
                   .planNode();
   testSerde(plan);
 }
 
+TEST_F(PlanNodeSerdeTest, writeWithAggregation) {
+  auto rowTypePtr = ROW({"c0", "c1", "c2"}, {BIGINT(), BOOLEAN(), VARBINARY()});
+  auto planBuilder =
+      PlanBuilder(pool_.get()).tableScan(rowTypePtr, {"c1 = true"}, "c0 < 100");
+  auto plan =
+      planBuilder
+          .tableWrite(
+              "targetDirectory", dwio::common::FileFormat::DWRF, {"min(c0)"})
```

Comment:
Would you also test with multiple stats over multiple columns?

Is it valid to collect difference set of stats for different columns?

Should we instead specify list of stats as aggregate function names and list of columns?

```
{"min", "max"}, {"c0", "c1", "c2"}
```

Reply:
For different columns, its supported aggregations are different, so we can't collect a set of stat for different columns.

I just update the code to cover more aggregation types